### ISSUE TITLE
[BUG] Header - nav tags need to have unique names

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#159]
**Describe the bug**
The header fails axe auto accessibility test for having navigation landmarks with no unique name. Landmarks must have a unique aria-label, aria-labelledby to make landmarks distinguishable and identify purpose.

**Expected behaviour**
Any landmarks that are repeated must have a unique name that clearly describes it purpose.

My thoughts are the header menu items could be in a nav with an accessible name of site navigation and the admiralty-header-profile could have the name profile.

Although the 'nav' is intended only for a major block of navigation links not for singular links so when the user has yet to sign in the Sign In link does not need to be in a 'nav' or would need a different unique name to the suggestion above for the admiralty-header-profile element.

Also I don't think the Admiralty logo link should be in the same nav as the header menu items as it may have a different purpose.

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/1894a050-baed-4ab0-ab51-76093fd60c36)
